spec/requests/errors_spec.rb

```diff
@@ -3,9 +3,15 @@
 require 'rails_helper'
 
 RSpec.describe 'Errors', type: :request do
+  let(:mock_user) { 'patron2' }
+
+  before do
+    login_permanently_as username: 'PATRON2', patron_key: mock_user
```

Comment:
needed this added, all specs were failing because it couldn't make `patron_path(patron)` execute without it